### PR TITLE
feat: add seo text on the bottom page

### LINKED ISSUE #292: Add SEO Text in the Footer
## Description

Refer to [Docs](https://docs.google.com/document/d/1im7CF8XaF4yYQ_jJjjW7dOKZz8tJFvidtW7bOlqt1Y8/edit?usp=sharing), section 1.2.2

To support the content for City Pages. We suggest adding more short text inside the page (can be put in bottom) to make the page more relevant & more prefer to be indexed.

- Provinces Homepage: https://www.wargabantuwarga.com/provinces

```
Cari & Temukan Informasi Fasilitas Kesehatan (Faskes) & Alat Kesehatan (Alkes) untuk COVID-19 di wilayah berikut {month} {year}.
```

- City Detail: https://www.wargabantuwarga.com/provinces/dkijakarta-1

```
List Informasi Fasilitas Kesehatan (Faskes) & Alat Kesehatan (Alkes) untuk COVID-19 di {City Name} {Date} {Month} {Year}.
```

## Tasks

- [ ] Add new small text on the Bottom of pages
- [ ] Screenshot and attach the changes

### PR DESCRIPTION
**Pull request by mazipan**
Closes #292

## Description

Add SEO Text on the bottom section to boost the content

Screen Shot 2021-07-22 at 09 45 04

Screen Shot 2021-07-22 at 09 45 16

### CODE REVIEW

File: pages/provinces/[provinceSlug]/index.tsx [hunk 1]
```diff
@@ -110,6 +116,15 @@ export default function ProvincePage(props: ProvinceProps) {
             isLoading={isLoading}
             provinceSlug={provinceSlug}
           />
+          <SeoText
+            textNode={
+              <span>
+                Daftar Informasi Fasilitas Kesehatan (Faskes) & Alat Kesehatan
+                (Alkes) untuk COVID-19 di {provinceName} per {getCurrentDate()}{" "}
+                {getCurrentMonth()} {getCurrentYear()}.
+              </span>
```
Comment (ekamuktia) [inline pages/provinces/[provinceSlug]/index.tsx:125]: How about using `<p>` tag instead of `<span>`?

File: lib/date-utils.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  return monthNames[monthIndex];
+}
+
+export function getCurrentYear(): number {
+  return new Date().getFullYear();
+}
+
+export function getCurrentDate(): number {
+  return new Date().getDate();
+}
```
Comment (zainfathoni) [inline lib/date-utils.ts:16]: Can we use `Date.prototype.toLocaleString()` instead for this? It should have been supported in all major browsers nowadays, shouldn't it?

I'm also using it at [the home page](https://github.com/kawalcovid19/wargabantuwarga.com/blob/ba189537c16236765df08078b9d154aafcd27c2f/pages/index.tsx#L44).

That way, I think we can simplify the implementation of the functions in this `date-utils.ts` file.

**Review by zainfathoni — CHANGES_REQUESTED** [review]
Another change request for simplifying our `date-utils.ts` implementation.

File: lib/date-utils.ts (hunk 2 above)
Comment (mazipan) [reply lib/date-utils.ts:32]: Done

File: pages/provinces/[provinceSlug]/index.tsx (hunk 1 above)
Comment (mazipan) [reply pages/provinces/[provinceSlug]/index.tsx]: Done

**Review by zainfathoni — APPROVED** [review]
LGTM 💯 
Thanks, Mas @mazipan!

**Pull request merged**